client/blocks/stats-sparkline/style.scss

```diff
@@ -0,0 +1,16 @@
+.stats-sparkline {
+	padding: 5px 0 2px;
+}
+
+.stats-sparkline__bar {
+	display: inline-block;
+	background-color: var( --color-sidebar-gridicon-fill );
```

Comment:
Good choice of color 👍  

Before:
![](https://cldup.com/pPk_amCLbZ.png)

After:
![](https://cldup.com/PcXwNkax8J.png)

While it differs from production, I would also use the same color as the icon on that menu item 👍